src/plugins/irc-events/list.js

```diff
@@ -5,7 +5,7 @@ var Msg = require("../../models/msg");
 
 module.exports = function(irc, network) {
 	var client = this;
-	var MAX_CHANS = 1000;
+	var MAX_CHANS = 500;
```

Comment:
A couple ideas for improvement of this, but **not necessarily for this PR**:

- *Easy*: Load 501 chans and in the front-end add a note saying "Only the 500 most crowded channels were displayed" if there is more than 500 chans. It's a cheap trick to avoid having an extra property `hasMoreThanFiveHundredChannels` 😄

- *Hard*: Load the first 100 only and add a "Show more channels" at the bottom or make it infinite scrolling that loads N new channels as you scroll. There is a mention of this pattern somewhere (I forget which issue/PR) for something like this: when opening a channel with lots of unread messages, ideally we would display the first unread ones and load new ones as we scroll down. These could use the same mechanism, but it's obviously much harder than the cheap solution above.